### ISSUE TITLE
incompatibilty with firefox 56 / doesnt show the dom 

### ISSUE THREAD

**Issue by Laminemett1973** [pierretholence/firepath#63]
incompatible with Firefox when update to v 56

**Comment by dhapolapankaj** [comment]
I am also facing the same issue.
Can we please have a look at this with highest priority?
Firefox version - 56.0.1
Firebug version - 2.0.19
Firepath version - 0.9.7.1.1

Regards
PD
![issuewithfirepath](https://user-images.githubusercontent.com/29034223/32145210-a0c590d2-bcea-11e7-9662-78d3b44dc539.png)

**Comment by dhapolapankaj** [comment]
@pierretholence 
Sorry to tag here. I would like to emphasize that there is no tool like FirePath. The DOM visualizer while creating XPath is the best feature, which is not available in any other tool.
I tried many other Xpath tools both in chrome and firefox.

Thanks
PD

**Comment by Muthuvignesh1991** [comment]
Agreed, as far as i have explored no other tool can replace Firepath
